ui_framework/components/form/_color_picker.scss

```diff
@@ -0,0 +1,29 @@
+.kuiColorSwatch {
+  width: 19px;
+  height: 19px;
```

Comment:
A var like this would go inside the `_index.scss` file in this component's directory, before the imports:

```scss
$colorPickerSize: 20px;

@import "assisted_input";
@import "check_box";
@import "label";
@import "search_input";
@import "select";
@import "static_input";
@import "text_area";
@import "text_input";
@import "color_picker";
```